worker/src/main/java/org/apache/celeborn/service/deploy/worker/memory/MemoryManager.java

```diff
@@ -255,12 +255,13 @@ protected void switchServingState() {
     switch (servingState) {
       case PUSH_PAUSED:
         pausePushDataCounter.increment();
-        logger.info("Trigger action: PAUSE PUSH, RESUME REPLICATE");
         if (lastState == ServingState.PUSH_AND_REPLICATE_PAUSED) {
+          logger.info("Trigger action: RESUME REPLICATE");
```

Comment:
I think the logs can be more precise like:
`Trigger action: PUSH_AND_REPLICATE_PAUSED -> PUSH_PAUSED`

Reply:
+1 with last serving state -> current serving state.

Reply:
And the log should be like "State changed: xxx -> yyy" to distinguish from the "Trigger action: Trim"

Reply:
In the previous code, including the log information you mentioned.

![image](https://github.com/apache/incubator-celeborn/assets/14961757/67e50a60-07dd-4ab0-8ce7-39624ef08100)

Reply:
At the same time, I think the logs related to trigger action should be kept, because it can be seen which actions are performed in the state switching